src/api.ts

```diff
@@ -0,0 +1,63 @@
+interface Project {
+  projectName: string;
+  techStack: string;
+  isCompleted: boolean;
+}
+
+interface Developer {
+  id: number;
+  name: string;
+  age: number;
+  isEmployed: boolean;
+  skills: Array<string>;
+  projects: Array<Project>;
+  experience: number;
+}
+
+const developers: Developer[] = [];
+
+function addDeveloper(developer: Developer): void {
```

Comment:
- remove unnecessary comments
- use the same format for error messages

Reply:
> - remove unnecessary comments
I removed this comments in the `addSkills` pull request, should I remove them in this branch too?

Reply:
no it's fine, 

but why are there duplicate commits in every PR?